content/agent/proxy.md

```diff
@@ -444,6 +444,97 @@ To verify that everything is working properly, review the HAProxy statistics at
 {{% /tab %}}
 {{< /tabs >}}
 
+## Using NGINX as a Proxy
+
+[NGINX][3] is a web server which can also be used as a reverse proxy, load balancer, mail proxy, and HTTP cache. You can use NGINX as a Proxy to send logs to Datadog:
+
+`agent ---> nginx ---> Datadog` or  
+`external log shipper ---> nginx ---> Datadog`
+
+### Proxy forwarding with NGINX
+#### NGINX configuration
+
+This example `nginx.conf` can be used to proxy logs to the Datadog intake. It does TLS wrapping to ensure internal plaintext logs are encrypted between your proxy and Datadog's log intake API endpoint:
+
+```
+user nginx;
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid; 
+
+include /usr/share/nginx/modules/*.conf;
+events {
+    worker_connections 1024;
+}
+# TCP Proxy for Datadog logs
+stream {
+    server {
+        listen 10514; #proxy listen port
+        proxy_ssl on;
+        proxy_pass agent-intake.logs.datadoghq.com:10516;
+    }
+}
+```
+
+#### Shipper configuration
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+To use the Datadog Agent as the logs collector, instruct the Agent to use the newly created proxy instead of establishing a connection directly with the logs intake by updating `datadog.yaml`:
+
+```yaml
+logs_config:
+  dd_url: myProxyServer.myDomain
+  dd_port: 10514
+  dev_mode_use_proto: false
+  dev_mode_no_ssl: true
+```
+
+The `dev_mode_no_ssl` parameter is set to `true` because establishing the SSL/TLS connection is handled by NGINX's `proxy_ssl on` option. **Note**: Set this option to `false` if you don't intend to use a proxy which can encrypt the connection to the logs intake.
```

Comment:
Possibly
```suggestion
The `dev_mode_no_ssl` parameter is set to `true` as the SSL/TLS connection is established via NGINX's `proxy_ssl on` option. **Note**: Set this option to `false` if you don't intend to use a proxy which can encrypt the connection to the logs intake.
```

Reply:
Checking with @l0k0ms to see if `via` should be avoided for translation purposes.